### ISSUE TITLE
[Question]: UIE多卡训练报错问题

### ISSUE THREAD

**Issue by fuqiang-benz** [PaddlePaddle/PaddleNLP#4610]
### 请提出你的问题

python -m paddle.distributed.launch --nproc_per_node=3 --backend=gloo  ./finetune.py --device cpu --logging_steps 10 --save_steps 100 --eval_steps 100 --seed 1000 --output_dir ./checkpoint/model_best --per_device_train_batch_size 16 --per_device_eval_batch_size 16 --num_train_epochs 20 --learning_rate 1e-05 --do_train --do_eval --do_export --overwrite_output_dir  --disable_tqdm True --metric_for_best_model eval_f1 --load_best_model_at_end True --save_total_limit 1

**Comment by fuqiang-benz** [comment]
> 

我用的是uie的信息抽取功能 自定义训练的时候 年前还可以用 paddlenlp升级到2.5就不可以用了 使用命令启动之后 会一直启动进程 起好几十个进程 直到内存排满 然后服务器死机
